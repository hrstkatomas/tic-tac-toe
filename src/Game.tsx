import React from 'react'


const Game: React.FC = () => {
  return (
    <div>
      Hello world
    </div>
  )
}

export default Game
