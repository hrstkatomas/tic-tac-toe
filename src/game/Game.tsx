import React from 'react'
import Board from './Board'

const Game: React.FC = () => {
  return (
    <div>
      <Board />
    </div>
  )
}

export default Game
