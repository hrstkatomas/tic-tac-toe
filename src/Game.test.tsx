import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import Game from './Game'

describe("Game component", () => {
  it('renders without crashing', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Game />, div)
    ReactDOM.unmountComponentAtNode(div)
  })
  
  it('renders according to snapshot', () => {
    const tree = renderer
      .create(<Game />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
