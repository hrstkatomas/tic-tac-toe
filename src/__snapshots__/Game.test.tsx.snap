// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game component renders according to snapshot 1`] = `
<div>
  Hello world
</div>
`;
